From: Alexey Dobriyan <adobriyan@openvz.org>
Date: Tue, 4 Mar 2008 12:33:53 +0000 (+0300)
Subject: CPT: select PM as well
X-Git-Url: http://git.openvz.org/?p=linux-2.6.24-openvz;a=commitdiff_plain;h=15938557b7ee1cebc893c676e0574a990fb79947;hp=1cfe443dac8890ab8e6e3eb8824b1aebb126f697

CPT: select PM as well

Just PM_SLEEP is not enough, because selecting logic doesn't select second and
all other levels of dependencies (which is probably a bug in build system
depending on how to look at it).
---

diff --git a/kernel/Kconfig.openvz b/kernel/Kconfig.openvz
index 5cdcdd3..5d5d6f0 100644
--- a/kernel/Kconfig.openvz
+++ b/kernel/Kconfig.openvz
@@ -70,6 +70,7 @@ config VZ_WDOG
 config VZ_CHECKPOINT
  	tristate "Checkpointing & restoring Virtual Environments"
  	depends on VE_CALLS && INET
+	select PM
 	select PM_SLEEP
 	select TUN
 	select VE_ETHDEV
